dist/api_data.py

```diff
@@ -754,6 +754,13 @@ def __cmp__(self, other):
         type='boolean'),
 ]),
 'WT_SESSION.strerror' : Method([]),
+'WT_SESSION.transaction_sync' : Method([
+    Config('timeout', '0', r'''
+        maximum amount of time to wait for background sync in seconds.
```

Comment:
1 second could be a long time to wait for a sync -- maybe in milliseconds or microseconds?